models/models.js

```diff
@@ -105,10 +113,18 @@ const Manga = types.model('Manga', {
 const Chapter = types.model('Chapter', {
   link: types.identifier,
   title: types.string,
+  date: types.maybeNull(types.Date),
```

Comment:
comment on why it's `maybeNull`: that it can be non-existent or invalid

Relevant note is that I lost data before when MST was unable to serialize as date, and then errored out when it tried to load the data 😞 
Would be good if MST or `mst-persist` could partially load data and skip entries with errors